src/pygenesis/http_helper.py

```diff
@@ -74,14 +75,220 @@ def get_data_from_endpoint(endpoint: str, method: str, params: dict) -> str:
     )
 
     response = requests.get(url, params=params_, timeout=(5, 15))
-    response.encoding = "UTF-8"
 
+    # if the response requires starting a job, the user is prompted to decide
+    try:
+        response_status_code = response.json().get("Status").get("Code")
+        if response_status_code == 98:
+            new_params = _jobs_params(params)
+
+            # start job if decided by user input
+            if type(new_params) == dict:
+                jobs_response = get_data_from_endpoint(
+                    endpoint=endpoint, method=method, params=new_params
+                )
+                # jobs_response = load_data(endpoint=endpoint, method=method, params=new_params, as_json=True)
+                jobs_catalogue_params, job_id = _jobs_job_id(
+                    jobs_response, params
+                )
+                response = json.loads(
+                    _jobs_catalogue_process(jobs_catalogue_params, job_id)
+                )
+                print(response)
+    except json.decoder.JSONDecodeError:
+        pass
+
+    response.encoding = "UTF-8"
+    print(response.json())
     _check_invalid_status_code(response.status_code)
     _check_invalid_destatis_status_code(response)
 
     return str(response.text)
 
 
+# TODO: test (Marco)
+def _generic_status_dict(
+    status_code: int, status_content: str, status_type: str
+) -> requests.Response:
+    """
+    Helper method which creates generic status dict.
+
+    Args:
+        status_code (int): Status code of the response
+        status_content (str): Status content of the response
+        status_type (str): Status type of the response (Information/ Warnung/ Fehler)
+
+    Returns:
+        requests.Response: the response from Destatis
+    """
+    # set up personalized requests.Reponse
+    request_status = requests.Response()
+    request_status.status_code = 42
+
+    # define status warning and texts
+    status_dict = {
+        "Status": {
+            "Code": status_code,
+            "Content": status_content,
+            "Type": status_type,
+        },
+    }
+
+    request_status._content = json.dumps(status_dict).encode("utf-8")
+
+    return request_status
+
+
+def _jobs_params(params: dict) -> dict:
+    """
+    Helper method which handles too large data requests with option of starting a job.
+
+    Args:
+        params (dict): dictionary of query parameters
+
+    Returns:
+        dict: new dict to start a job
```

Comment:
Würde hier "new request params dict to start a job" schreiben, damit klar ist, wofür das dict verwendet werden soll